submit-web/src/models/Submission.ts

```diff
@@ -55,3 +62,15 @@ export type Submission = {
   created_date: string;
   submitted_by: string;
 };
+
+export type StaffSubmission = Submission & {
```

Comment:
It's not a  submission, that naming should be reserved for proponents submitting management plans or consultation records or stuff